source/public/styles/styles2.css

```diff
@@ -0,0 +1,415 @@
+
+/*Styling for Home Page!*/
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+*, *::before, *::after{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0; 
+    font-family: 'Inter', "FontAwesome", sans-serif;
+    
+}
+
+#homepage {
+    height:15%;
+    width:85%; /*Percentage based on parent container size*/
+    margin: 0 auto; /*Sets top and bottom margin to 0. but horizontal auto adjusts according to browser*/
+    display:flex;
+    align-items: center; /*Aligns content vertically in the center of the container*/
+    justify-content: center; /*Aligns flex items horizontally*/
+}
+
+#homepage h1{
+    color: #FF9B83;
+}
+
+main{
+    display:flex;
+    flex-grow: 1;
+    justify-content: space-evenly;
+}
+
+#calendarContainer{
+    display:flex;
+    justify-content: flex-start;
+    border: 1px solid white;
+    width: 50%;
+    border-radius: 10px;
+}
+
+.calendar{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: white;
+    gap: 20px;
+    border-radius: 10px;
+    
+    
+}
+
+.days{
+    display: flex;
+    width: 97%;
+    margin: 0 auto;
+    justify-content: space-around;
+    border-bottom: 1px solid black;
+    font-weight: bold;
+    background-color: white;
+    padding-bottom:2%;
+    
+}
+
+.day{
+    background-color:white;
+}
+
+.dates{
+    display:grid;
+    grid-template-columns: 1fr;
+    width: 100%;
+   
+    height:100%;
+    
+}
+
+.highlight{
+    background-color: #FFC700 !important;
+    border-radius: 100%;
+
+}
+
+.row{
+    display:grid;
+    grid-template-columns: repeat(7,1fr);
+    a{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-decoration: none;
+    }
+}
+
+.date{
+    background-color: white;
+
+    border:transparent;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    padding: 20px;
+    font-size: 1.2rem;
+    width: 50%;
+    height:50%;
+    margin:auto;
+    
+
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    background: #FFF7EC;
+}
+
+/*.header{
+    display:flex;
+    width:100%;
+    justify-content: center;
+    
+}*/
+.header {
+    
+    height:10%;
+    width: 85%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between; 
+    align-items: center;
+    background-color: white;
+    padding-top: 5%;
+
+}
+
+.headbuttons{
+    display: flex;
+    width:50%;
+    justify-content: space-between;
+    align-items: center;
+    
+}
+.dummyy{
+    display: flex;
+    flex-shrink:1;
+    
+   
+}
+
+.month-year{
+    color: #685489;
+    font-weight:bold;
+    font-size: 2rem;
+    background-color: white;
+    
+
+}
+
+#home-today{
```

Comment:
When the stylesheets are combined this rule should be unnecessary (it'll use the same styling as the button on the journal page)